projenrc/copy-cli-version-to-assembly.task.ts

```diff
@@ -0,0 +1,24 @@
+import { promises as fs } from 'fs';
```

Comment:
just FYI, not sure which is better
```suggestion
import * fs from 'fs/promises';
```